ansible/roles/cartodb.postgresql/tasks/main.yml

```diff
@@ -0,0 +1,57 @@
+---
+
+ - name: Add cartodb postgresql apt repository
+   apt_repository: repo=ppa:cartodb/postgresql-9.3 update_cache=yes
+
+ - name: Install postgresql client packages
+   apt: name={{ item }} state=present
+   with_items:
+    - libpq5
+    - libpq-dev
+    - python-psycopg2
+    - postgresql-client-{{ postgresql_version }}
+    - postgresql-client-common
+
+ - name: Install postgresql server packages
+   apt: name={{ item }} state=present
+   with_items:
+    - postgresql-{{ postgresql_version }}
+    - postgresql-contrib-{{ postgresql_version }}
+    - postgresql-server-dev-{{ postgresql_version }}
+    - postgresql-plpython-{{ postgresql_version }}
+
+ - name: Add CartoDB pg-schema-trigger repo
+   apt_repository: repo=ppa:cartodb/pg-schema-trigger state=present update_cache=yes
+
+ - name: Install postgresql schema triggers
+   apt: name=postgresql-9.3-pg-schema-triggers state=present
+
+ - name: Configure PostgreSQL host-based authentication
+   template: src=pg_hba.conf.j2 dest=/etc/postgresql/{{ postgresql_version }}/main/pg_hba.conf
+
+ # Need to force a restart now so that auth settings take effect for all future tasks
+ #   Otherwise, things like `bundle exec rake db:create` will hang asking for a password
+ # Ansible docs say that 'These notify actions are triggered at the end of each block of tasks
+ #   in a playbook', which seems like they would trigger at the end of the cartodb.postgresql role,
+ #   but they don't seem to trigger until at the very end of provisioning, which is too late
+ #   for this particular handler.
+ - name: Restart PostgreSQL
+   service: name=postgresql state=restarted
```

Comment:
This is probably the best approach given you know exactly what needs to be restarted, but also though I would mention `flush_handlers` as a way to force execution of the notifications that have been queued up to that point:

```yaml
- meta: flush_handlers
```